pkg/operator/ceph/csi/version.go

```diff
@@ -0,0 +1,110 @@
+/*
+Copyright 2020 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package csi
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+
+	"github.com/pkg/errors"
+)
+
+// CephCSIVersion represents the Ceph CSI version format
+type CephCSIVersion struct {
+	Major  int
+	Minor  int
+	Bugfix int
+}
+
+var (
+	//minimum supported version is 2.0.0
+	minimum = CephCSIVersion{2, 0, 0}
+
+	//supportedCSIVersions are versions that rook supports
+	supportedCSIVersions = []CephCSIVersion{minimum}
+
+	// for parsing the output of `cephcsi`
+	versionCSIPattern = regexp.MustCompile(`v(\d+)\.(\d+)\.(\d+)`)
+)
+
+func (v *CephCSIVersion) String() string {
+	return fmt.Sprintf("%d.%d.%d",
```

Comment:
are we  missing `v` here or else it will be `2.0.0`

Reply:
You are correct. Should be there, too.